sdk/core/core-rest-pipeline/src/nodeHttpClient.ts

```diff
@@ -337,7 +338,7 @@ function getDecodedResponseStream(
   return stream;
 }
 
-function streamToText(stream: NodeJS.ReadableStream): Promise<string> {
+function streamToText(stream: NodeJS.ReadableStream, encoding: "utf8" | "binary"): Promise<string> {
```

Comment:
It doesn't make sense for `streamToText` api to return binary data. It should return text

Reply:
I didn't mean to ship this, just suggesting what needs to be done in terms of the encoding.